client/components/PubEdgeListing/PubEdgeListing.js

```diff
@@ -0,0 +1,138 @@
+import { NonIdealState } from '@blueprintjs/core';
+import PropTypes from 'prop-types';
+import React, { useCallback, useState } from 'react';
+
+import { Filter, Mode, allFilters } from './constants';
+import { pubEdgeType } from '../PubEdge/constants';
+import PubEdgeListingCard from './PubEdgeListingCard';
+import PubEdgeListingCounter from './PubEdgeListingCounter';
+import PubEdgeListingControls from './PubEdgeListingControls';
+
+require('./pubEdgeListing.scss');
+
+const propTypes = {
+	accentColor: PropTypes.string.isRequired,
+	minimal: PropTypes.bool,
+	pubTitle: PropTypes.string,
+	pubEdges: PropTypes.arrayOf(pubEdgeType).isRequired,
+};
+
+const defaultProps = {
+	minimal: false,
+	pubTitle: '',
+};
+
+const filterPubEdges = (filters, pubEdges) =>
+	pubEdges.filter((pubEdge) => {
+		let result = false;
+
+		if (filters.indexOf(Filter.Parent) > -1) {
+			result = result || pubEdge.pubIsParent;
+		}
+
+		if (filters.indexOf(Filter.Child) > -1) {
+			result = result || !pubEdge.pubIsParent;
+		}
+
+		return result;
+	});
```

Comment:
This is placeholder code that will likely change once we get the views hooked up to the API.